src/opm/parser/eclipse/EclipseState/Schedule/Action/Enums.cpp

```diff
@@ -0,0 +1,26 @@
+/*
+  Copyright 2021 Equinor ASA.
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+namespace Opm {
+namespace Action {
+
+#include <opm/parser/eclipse/EclipseState/Schedule/Action/Enums.hpp>
```

Comment:
will this get code later? as is this cpp file is pointless.

Reply:
Will get code yes.